source/common/access_log/access_log_formatter.cc

```diff
@@ -850,6 +857,39 @@ ResponseTrailerFormatter::formatValue(const Http::RequestHeaderMap&, const Http:
   return HeaderFormatter::formatValue(response_trailers);
 }
 
+GrpcStatusFormatter::GrpcStatusFormatter(const std::string& main_header,
+                                         const std::string& alternative_header,
+                                         absl::optional<size_t> max_length)
+    : HeaderFormatter(main_header, alternative_header, max_length) {}
+
+std::string GrpcStatusFormatter::format(const Http::RequestHeaderMap&,
+                                        const Http::ResponseHeaderMap&,
+                                        const Http::ResponseTrailerMap& response_trailers,
+                                        const StreamInfo::StreamInfo&) const {
+  const auto grpc_status_code_str = HeaderFormatter::format(response_trailers);
```

Comment:
`grpc-status` can also appear in the response headers (when the error happens before any messages has been sent), so i'd expect this operator to handle that